test/fileSummaryV1/fileSummaryV1.test.js

```diff
@@ -32,25 +33,36 @@ const mockResponse = () => {
     };
 };
 
+// helpers.convertToFileInfoResponse (contentType, size, timeCreated, updated, md5Hash, bucket, name, gsUri, signedUrl)
```

Comment:
What is this line? Docs? Orphaned code?

Reply:
good catch — that was a leftover reminder